common/payment/definition.proto

```diff
@@ -10,11 +10,4 @@ enum PaymentProviderId {
     CYBERSOURCE = 0;
```

Comment:
are these used somewhere?

Reply:
They shouldn't be used anywhere.  If they are then its a mistake.

Reply:
Still pushing updates i'm finding - i have removed `PaymentProviderId`

Slightly related:
The `saga.api.myth` messages seem to involve some payment and orders data. ie: 
[QuoteBuyingMythTokenRequest](https://github.com/MythicalGames/saga-sdk-proto/blob/develop/api/myth/definition.proto#L41) from `saga.api.myth` is importing `saga.api.order` for the use of`PaymentProviderData`.

I'd ideally like to remove all payment and order dependencies and just move them into `saga.api.myth`. Thoughts?

Please contact me if this is confusing.

Reply:
yeah lets move it to the nmyth dir for now if they need it.

Reply:
Ok. The directory and messages are all named `myth` rather than `nmyth`. I can rename to nmyth is that's whats actually being referred to, not totally sure the intention here.